yaks/README.md

```diff
@@ -665,11 +678,42 @@ yaks.call(array_of_widgets, mapper: MyCollectionMapper, item_mapper: WidgetMappe
 ```
 
 If the mapper is left unspecified, Yaks will inspect whatever you pass
-it, and try various constant lookups based on naming. These all happen
+it. First it will test the given object against the mapping defined using `mapper_for`.
+If no mapper is found, it wil call `derive_mapper_from_item` or `derive_mapper_from_collection`
```

Comment:
Also, it would be great if you described somewhere that `derive_mapper_from_object` calls `deriver_mapper_from_item` or `derive_mapper_from_collection`. Just so that users understand what is better.